### ISSUE TITLE
The training and testing of segmentation model

### ISSUE THREAD

**Issue by DaiZhewei** [sjtuplayer/anomalydiffusion#11]
Hi author, I would like to ask two questions:
1. whether the method is divided into 15 classes to train the classification model and segmentation model respectively
2. how to divide the training set and test set？
Looking forward to your answer, thanks!

**Comment by sjtuplayer** [comment]
1. Yes, we train one classification model and segmentation model for each sample class (e.g., bottle, capsule) separately.
2. The training set is the generated data while the test data is the last 2/3 data in the test set of MvTec.

**Comment by sjtuplayer** [comment]
> Hi author, I would like to ask two questions:
> 
> 1. whether the method is divided into 15 classes to train the classification model and segmentation model respectively
> 2. how to divide the training set and test set？
>    Looking forward to your answer, thanks!

You can refer to ''train-unet.py", "test-unet.py" and "train-classification.py" for more details

**Comment by DaiZhewei** [comment]
thanks, one more question, did you use all the masks in the test set when training the TI or only the first 1/3 of the masks？

**Comment by DaiZhewei** [comment]
> @DaiZhewei could you send your email address we may discuss some things,
hi, I e-mailed the e-mail address on your home page.

**Comment by CVKim** [comment]
@sjtuplayer 

Hi author,

It seems that the file train-unet.py is missing. Could it have been removed?

**Comment by sjtuplayer** [comment]
> @sjtuplayer
> 
> Hi author,
> 
> It seems that the file train-unet.py is missing. Could it have been removed?

Hi. It is train-localization.py in the current version.

**Comment by CVKim** [comment]
In the "Anomaly Diffusion" paper, is it correct to say that the performance evaluation focused more on anomaly detection, specifically localization and classification, rather than on segmentation?

Test-localization and test-classification.

![image](https://github.com/user-attachments/assets/1360c235-f140-45f7-a566-13691ee83e11)
